src/integrationtests/java/com/aws/greengrass/integrationtests/ShadowManagerTest.java

```diff
@@ -171,4 +177,34 @@ void GIVEN_existing_sync_information_WHEN_config_updates_THEN_removed_sync_infor
                 new Pair<>(THING_NAME2, "Shadow-0"),
                 new Pair<>(THING_NAME2, "Shadow-5")));
     }
+
+
+    @Test
+    @SuppressWarnings("PMD.CloseResource")
+    void GIVEN_shadow_manager_WHEN_individual_config_resets_THEN_respond_to_config_updates(ExtensionContext context)
+            throws Exception {
+        ignoreExceptionOfType(context, SkipSyncRequestException.class);
+        MqttClient mqttClient = mock(MqttClient.class);
```

Comment:
Note for the future: (maybe add a TODO) - We should use Fakes/Test Doubles for integration tests instead of mocking ( I know the test above is doing the same)